taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -912,6 +985,7 @@ export class VsCodeHelper {
 	private dataProviders: { refresh: () => void }[] = [];
 	private contractsDataProvider?: ContractsDataProvider;
 	private sandboxesDataProvider?: SandboxesDataProvider;
+	private testDataProvider?: TestDataProvider;
```

Comment:
Why do we make this field optional? Don't we always assign a value to it in the constructor?

Reply:
Not in the constructor, but when the method `registerDataProviders` is called. If we make it required, static analysis will complain that it has no value when the class is instantiated (either direct value when defining field or in constructor).